### PR TITLE
Clarify function docs

### PR DESCRIPTION
**Pull request by cmeeren**
I find these descriptions a bit more clear. But please check for errors; I may have misunderstood some details.

By the way, how do both of these functions work with tuples? Perhaps that should be mentioned, too?

### CODE REVIEW

**Comment by cmeeren** [comment]
Sorry for the rogue EOL edit; the GitHub editor threw that in for free...

File: src/Fable.React.Helpers.fs [hunk 1]
```diff
@@ -202,8 +202,9 @@ module Helpers =
         false
 #endif
 
-    /// Same as the default `React.memo` equality but ignores functions.
-    /// Useful in combination with `FunctionComponent.Of` `memoizeWith` parameter when F# equality isn't required
+    /// Memberwise referential comparison (same as default React.memo) that ignores
```
Comment (alfonsogarciacaro) [inline src/Fable.React.Helpers.fs]: I agree the comment is not very clear but the tricky part is this is not exactly referential comparison (for numbers and strings for example the comparison is by value). There's some [discussion about it here](https://github.com/fable-compiler/fable-react/pull/153#discussion_r276569792).

Comment (cmeeren) [reply src/Fable.React.Helpers.fs]: Better now? Note that I use the term "value types", but I'm not sure exactly how the equality works with structs (e.g. `DateTime`) in the JS environment. Are they compared by value?

**Comment by alfonsogarciacaro** [comment]
Thanks!

**Pull request merged**